b2/console_tool.py

```diff
@@ -1252,9 +1321,8 @@ def run(self, args):
 
     def get_policies_manager_from_args(self, args):
         if args.includeRegex and not args.excludeRegex:
-            raise CommandError(
-                '--includeRegex cannot be used without --excludeRegex at the same time'
-            )
+            logger.warning('--includeRegex is skipped without --excludeRegex')
```

Comment:
it has to be an error

Reply:
As discussed with @ppolewicz , I'm going to implement this assertion in SDK, not CLI.

Reply:
I've removed this log completely